### PR TITLE
Update mac os notifcations segment to work with sonoma

### PR DESCRIPTION
**Pull request by ryanfabela**
Also updated id for messages and added slack id to help out others

### CODE REVIEW

**Comment by ryanfabela** [comment]
tested this locally on Sonoma (Version 14.0 (23A344))

File: segments/macos_notification_count.sh [hunk 1]
```diff
@@ -1,16 +1,20 @@
 # Report macOS notification counts for given app ids (banner notifications only)
 # Based on http://www.ign.com/boards/threads/a-crumby-way-to-get-an-unread-count-of-imessages-into-applescript.453061379/
 
-TMUX_POWERLINE_SEG_MACOS_NOTIFICATION_COUNT_APPIDS_DEFAULT="5"
+# 118 = messages
+# 51 = slack 
+
+TMUX_POWERLINE_SEG_MACOS_NOTIFICATION_COUNT_APPIDS_DEFAULT="118 51"
```
Comment (erikw) [inline segments/macos_notification_count.sh:7]: Let’s not add Slack to the default, not everyone has this app.

**Review by erikw — CHANGES_REQUESTED** [review]
Thanks. Just one small fix

**Comment by ryanfabela** [comment]
Understoon. Makes total sense. Made the change, thank you

**Pull request merged**